### DISCUSSION TITLE
Priority Selection

### DISCUSSION THREAD

**Discussion by Dadadahhhhh** [roovo/obsidian-card-board#195] [Ideas]
What if you added an additional "Priority" setting  And the end of the Check box, When a user adds a p1, p2, or p3, that could place the task at the top of the kanban in order of priority...
![image](https://github.com/roovo/obsidian-card-board/assets/122122422/305658d4-efff-49e0-99f8-23c4cd768449)

**Comment by roovo** [comment]
Hi @Dadadahhhhh - thank you and I hear you!  I have had a few asks about handling priorities - and it is something that I would like to build in as I can definately see the use/helpfulness of this.

At the moment, there is no concept of priority in Cardboard, so this would need to be added (along with the ability to understand priorities in Tasks format).  I would like to start to think about sorting and ordering of tasks in columns once I have the current drag and drop work completed as it follows nicely from this!  So implementing something for dealing with this is reasonably high on the list of things I would like to get done :)
